### ISSUE TITLE
Build failure on debian 9

### ISSUE THREAD

**Issue by mgerhardy** [nextcloud/client_theming#68]
```
[ 13%] Built target nextclouddolphinpluginhelper
[ 14%] Built target nextclouddolphinactionplugin
[ 16%] Built target nextclouddolphinoverlayplugin
[ 17%] Building CXX object src/libsync/CMakeFiles/nextcloudsync.dir/theme.cpp.o
[ 18%] Building CXX object src/libsync/CMakeFiles/nextcloudsync.dir/__/3rdparty/certificates/p12topem.cpp.o
/home/mgerhardy/dev/client_theming/client/src/3rdparty/certificates/p12topem.cpp:68:5: error: use of undeclared identifier 'SSLeay_add_all_algorithms'
    SSLeay_add_all_algorithms();
    ^
1 error generated.
src/libsync/CMakeFiles/nextcloudsync.dir/build.make:950: recipe for target 'src/libsync/CMakeFiles/nextcloudsync.dir/__/3rdparty/certificates/p12topem.cpp.o' failed
make[2]: *** [src/libsync/CMakeFiles/nextcloudsync.dir/__/3rdparty/certificates/p12topem.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
/home/mgerhardy/dev/client_theming/client/src/libsync/theme.cpp:33:12: fatal error: '/home/mgerhardy/dev/client_theming/build-1/../nextcloudtheme/nextcloudtheme.h' file not
      found
#  include INCLUDE_FILE(THEME_INCLUDE)
           ^
/home/mgerhardy/dev/client_theming/client/src/libsync/theme.cpp:32:28: note: expanded from macro 'INCLUDE_FILE'
#  define INCLUDE_FILE(M)  QUOTEME(M)
                           ^
/home/mgerhardy/dev/client_theming/client/src/libsync/theme.cpp:31:28: note: expanded from macro 'QUOTEME'
#  define QUOTEME(M)       #M
                           ^
<scratch space>:49:1: note: expanded from here
"/home/mgerhardy/dev/client_theming/build-1/../nextcloudtheme/nextcloudtheme.h"
^
1 error generated.
src/libsync/CMakeFiles/nextcloudsync.dir/build.make:734: recipe for target 'src/libsync/CMakeFiles/nextcloudsync.dir/theme.cpp.o' failed
make[2]: *** [src/libsync/CMakeFiles/nextcloudsync.dir/theme.cpp.o] Error 1
CMakeFiles/Makefile2:242: recipe for target 'src/libsync/CMakeFiles/nextcloudsync.dir/all' failed
make[1]: *** [src/libsync/CMakeFiles/nextcloudsync.dir/all] Error 2
Makefile:149: recipe for target 'all' failed
make: *** [all] Error 2
mgerhardy@mattn-laptop:~/dev/client_theming/build-linux$
```

**Comment by mgerhardy** [comment]
related: #42

**Comment by mgerhardy** [comment]
libssl-dev was installed, but libssl1.0-dev is needed, after installing that and using the fixed cmake call from #62 it compiled.
